src/main/kotlin/gg/essential/gradle/util/RelocationTransform.kt

```diff
@@ -118,6 +118,8 @@ abstract class RelocationTransform : TransformAction<RelocationTransform.Paramet
                     val writer = ClassWriter(0)
                     reader.accept(ClassRemapper(KotlinMetadataRemappingClassVisitor(remapper, writer), remapper), 0)
                     writer.toByteArray()
+                } else if (entry.name.startsWith("META-INF/services/")) {
+                    String(originalBytes).replace('.', '/').lines().map(remapper::mapValue).joinToString("\n").replace('/','.').encodeToByteArray()
```

Comment:
Shouldn't this be using `remapper::map` rather than `mapValue` which will only map the string if the file has explicitly been included in `remapStringsIn`?

Reply:
Ah, yeah. I saw the mapvalue override in `ClassPrefixAndStringsRemapper` and just assumed mapvalue was the correct method to use.